### DISCUSSION TITLE
Help with understanding an error ('from' must be finite number)

### DISCUSSION THREAD

**Discussion by denizstudios** [facebookexperimental/Robyn#323] [General]
Hi, while building the model using real data i'm running into an error.
Not quite sure what I could investigate/fix to know why this is being caused. Any help is appreciated-thanks.
![image](https://user-images.githubusercontent.com/36769722/155572241-a0baba7c-2453-46ae-9b70-e3e54d2225e2.png)

**Comment by laresbernardo** [comment]
Hi @denizstudios 
There was a bug on `lambda_seg()` we fixed this morning. Please, can you update the package and restart your R session? Should be good now.  Read more here: #317

**Reply by denizstudios** [reply]
hi @laresbernardo , this indeed helped it - thanks! Back to training the model now :)
